learn/experimental/vector_search.mdx

```diff
@@ -19,6 +19,10 @@ If using Meilisearch Cloud, navigate to your project overview and find "Experime
 
 ![A section of the project overview interface titled "Experimental features". There are two options: "Score details" and "Vector store". "Vector store" is turned on.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vector-search/01-cloud-vector-store.png)
 
+<Capsule intent="note" title="Waiting list">
+To ensure proper onboarding and scaling of Meilisearch Cloud's latest AI-powered search offering, you must enter the waitlist before activating vector search.
```

Comment:
This is only for v1.7 onwards, previous versions from v1.3 to v1.6 don't have any embedder settings so you can just enable the experimental feature to use the vector store

Reply:
If I understand your comment correctly, I don't think there's much to change here. We only document the latest Meilisearch release, so describing the behaviour of previous releases is sort of out of scope.

Also is this tied to the engine version or to the Meilisearch Cloud version? We added embedder options on v1.6.

Reply:
Discussed this over slack and decided to keep it as is.